suite-common/invity/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+    "extends": "../../tsconfig.json",
+    "compilerOptions": {
+        "outDir": "libDev",
+        "rootDir": "../../../"
```

Comment:
why this `rootDir`?

Reply:
that was a miss 3a983db0f235c4cba5a8354683b4cdae0ad024eb